doc/src/records/rsys.md

```diff
@@ -274,6 +591,17 @@ For rsType=ASHP, auxiliary electric (“strip”) heating capacity. If AUTOSIZEd
   ----------- --------------------------- ------------- -------------- -----------------
   Btu/hr      *AUTOSIZE* or *x* $\ge$ 0   0             no             constant
 
+**rsAFUEAuxH=*float***
+
+For rsType=ASHP, auxiliary electric (“strip”) heating coefficient of performance. Energy use for auxiliary heat is accumulated to end use HPBU of meter rsElecMtr (that is, auxiliary heat is assumed to be electric).
```

Comment:
Replace "coefficient of performance" with "annualized fuel utilization efficiency"